pkg/controlplane/controller_test.go

```diff
@@ -34,567 +33,6 @@ import (
 	netutils "k8s.io/utils/net"
 )
 
-func TestReconcileEndpoints(t *testing.T) {
```

Comment:
Just to make sure I'm understanding this PR correctly, these tests were moved, not deleted, right?

Reply:
correct